it/spring/boot2-tomcat9/build.gradle

```diff
@@ -1,5 +1,6 @@
 dependencies {
     implementation project(':spring:boot2-starter')
+    implementation project(':spring:boot2-webflux-starter')
```

Comment:
Is this to make sure MVC is used when both starter-web and webflux-starter are present in the class path? I think this is somewhat confusing. How about adding a separate integration test module dedicated to such a scenario? e.g. `it/spring/boot2-mixed`?

Reply:
good idea. 👍

In addition to this, it seems necessary to remove the dependency of `boot2-webflux-autoconfigure` from `boot2-autoconfigure` and separate it into another module for testing.

e.g.
- `it/spring/boot2-mixed`
- `it/spring/boot2-mixed-tomcat9`

Reply:
SGTM. Please go ahead 🙇